build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'java'
```

Comment:
This sets up a standard way to compile Java, and a way to get any external libraries we want to use easily